test/json_converters/datetime_converter_test.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:data_class_plugin/public/json_converter.dart';
```

Comment:
Try to keep imports ordered (I will add a lint for that)
You can do that with the IDE as action